fpdf/fpdf.py

```diff
@@ -195,7 +196,7 @@ def __init__(
         Args:
             orientation (str): possible values are "portrait" (can be abbreviated "P")
                 or "landscape" (can be abbreviated "L"). Default to "portrait".
-            unit (str): possible values are "pt", "mm", "cm" or "in".
+            unit (str, int, float): possible values are "pt", "mm", "cm" or "in".
```

Comment:
Could you please explain in docstring how a numeral value is interpreted / used?

Reply:
Done